core/src/foundations/ionic.utility.scss

```diff
@@ -0,0 +1,115 @@
+/**
+ * Ionic Design System
+ */
+
+.color-primary-10 { color: #f7faff; }; .background-color-primary-10 {background-color: #f7faff;}
```

Comment:
~ We should identify if prettier can format this output. This could be a technical debt ticket/not blocking this PR.

Reply:
I already adjusted this on the script file and changed the `npm run tokens` script to also run `lint.sass.fix.` The generated file is different now.

Also, due to lint there were some errors that made me remove some prohibited properties (I added context on the PT description about this)